### ISSUE TITLE
Error when starting mylar after update

### ISSUE THREAD

**Issue by mwoods** [evilhero/mylar#1464]
I just updated Mylar and I get this:

C:\Mylar>Mylar.py
Traceback (most recent call last):
  File "C:\Mylar\Mylar.py", line 26, in <module>
    import mylar
  File "C:\Mylar\mylar\__init__.py", line 41, in <module>
    from mylar import logger, versioncheckit, rsscheckit, searchit, weeklypul
, dbupdater, PostProcessor, helpers, scheduler #versioncheck, rsscheck, searc
PostProcessor, weeklypull, helpers, scheduler
  File "C:\Mylar\mylar\rsscheckit.py", line 21, in <module>
    from mylar import logger, rsscheck, helpers, auth32p
  File "C:\Mylar\mylar\rsscheck.py", line 16, in <module>
    from mylar import db, logger, ftpsshup, helpers, auth32p, utorrent
  File "C:\Mylar\mylar\auth32p.py", line 7, in <module>
    from bs4 import BeautifulSoup
  File "C:\Mylar\bs4\__init__.py", line 29, in <module>

  File "C:\Mylar\bs4\builder\__init__.py", line 306, in <module>

  File "C:\Mylar\bs4\builder\_html5lib.py", line 57, in <module>
AttributeError: 'module' object has no attribute '_base'

C:\Mylar>

**Comment by evilhero** [comment]
Not sure how old your version was, bs4 was moved into the lib/ directory awhile ago, but it's trying to load the module from the old location. 

Delete all the .pyw files you have in the root of mylar and in the mylar/ subdirectory, make sure mylar is shutdown first, and then start it up and see.

**Comment by mwoods** [comment]
@evilhero  Thanks for responding so quickly. I looked in the directory(s) listed and I didn't see any .pyw files. I think I have an explanation on why there are phantom directories there that were moved a few versions back. There was a time where mylar wasn't working for me and I grabbed a version that was a bit older and moved it into the directory that wasn't working. By doing this, I'm sure I kept around a few files/directories that were no longer need. Now, I know how to move over my config and the mylar.db. 

I have grabbed -- source build 646309b8f64c560a8dccc5f1e499fdb2ff670b7e. from github and moved my DB and config file over. At first it said it was an unknown version and I updated it and now it says its fine. 

Things look like they are running properly. I consider this issue closed and thanks for your help!!
